LibreNMS/OS/TplinkEap.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * TplinkEap.php
+ *
+ * TP linke EAP (Omada)
+ *
+ * @link       https://www.librenms.org
+ *
+ * @copyright  
+ * @author     
+ */
+
+namespace LibreNMS\OS;
+
+use App\Models\Device;
+use LibreNMS\Device\WirelessSensor;
+use LibreNMS\Interfaces\Discovery\OSDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessClientsDiscovery;
+use LibreNMS\OS;
+
+class TplinkEap extends OS implements OSDiscovery,WirelessClientsDiscovery
+{
+   /**
+    * 
+    */ 
+    public function discoverOS(Device $device): void
+    {
+        $device->hardware = snmp_get($this->getDeviceArray(), '.1.3.6.1.2.1.1.5.0', '-Osqnv');
```

Comment:
Use discovery yaml instead:

```yaml
modules:
    os:
        sysDescr_regex: '/(?<version>.*)/'
        hardware: SNMPv2-MIB::sysName.0
```

Reply:
Well, this is not clear to me - from existing YAML it looks like that I cannot combine existing yaml with your suggestinon. 

this is not working for me: 

```
os: tplink-eap
modules:
    os:
        hardware: SNMPv2-MIB::sysName.0
        sysDescr_regex: '/(?<version>.*)/'
type: wireless
icon: tplink
over:
    - { graph: device_bits, text: 'Device Traffic'}
    - { graph: device_wireless_clients, text: 'Number of Clients' }      
poller_modules:
    wifi: true
discovery:
    - sysObjectID:
             .1.3.6.1.4.1.11863.3.
```

am i missing something?